services/server/server.go

```diff
@@ -596,6 +599,14 @@ func (s *Server) ChangeSettings(ctx context.Context, req *serverpb.ChangeSetting
 		return nil, err
 	}
 
+	if settings.SaaS.STTEnabled {
```

Comment:
it will start checks on each settings update, is it ok? Maybe it's better to start checks only if we have enable STT request?

Reply:
you are right, will change it